src/AdminAan/SFA.DAS.AdminAan/Application/CalendarEvents/Commands/Update/PutEventCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MediatR;
+using SFA.DAS.AdminAan.Application.CalendarEvents.Commands.Create;
+using SFA.DAS.AdminAan.Infrastructure;
+
+namespace SFA.DAS.AdminAan.Application.CalendarEvents.Commands.Update;
+
+public class PutEventCommandHandler : IRequestHandler<PutEventCommand, Unit>
+{
+    private readonly IAanHubRestApiClient _apiClient;
+
+    public PutEventCommandHandler(IAanHubRestApiClient apiClient)
+    {
+        _apiClient = apiClient;
+    }
+
+    public async Task<Unit> Handle(PutEventCommand command, CancellationToken cancellationToken)
+    {
+        var response = await _apiClient.PutCalendarEvent(command.RequestedByMemberId!, command.CalendarEventId!,
+            command, cancellationToken);
+
+        if (response.ResponseMessage.StatusCode == System.Net.HttpStatusCode.NoContent && command.Guests.Any())
```

Comment:
You have to assume that the status code is success here as otherwise an exception would have occurred. So no status check is necessary.

Reply:
ok, removed